### PR TITLE
Spline #330 Adding few improvements to the UI

### PR DESCRIPTION
**Pull request by leborgneromain**
Spline #330 Adding default values in dashboard

Spline  #330 if the name of the path is *, display the previous path.

### CODE REVIEW

File: consumer-services/src/main/scala/za/co/absa/spline/consumer/service/repo/ExecutionEventRepositoryImpl.scala [hunk 1]
```diff
@@ -38,11 +38,21 @@ class ExecutionEventRepositoryImpl @Autowired()(db: ArangoDatabaseAsync) extends
...
-        LET executionEvents = (
-                    FOR p IN progress
-                        FILTER p._creationTimestamp < TO_NUMBER(@asAtTime) && p.timestamp > TO_NUMBER(@timestampStart) && p.timestamp < TO_NUMBER(@timestampEnd)
-                        RETURN p
-                )
+        LET executionEventsFiltered = (
+              FOR p IN progress
+                FILTER p._creationTimestamp < TO_NUMBER(@asAtTime) && p.timestamp >= TO_NUMBER(@timestampStart) && p.timestamp <= TO_NUMBER(@timestampEnd)
+                RETURN p
+              )
```
Comment (wajda) [inline consumer-services/src/main/scala/za/co/absa/spline/consumer/service/repo/ExecutionEventRepositoryImpl.scala]: nit: indents

File: client-ui/src/app/components/dashboard/dashboard.component.ts [hunk 2]
```diff
@@ -87,26 +89,27 @@ export class DashboardComponent implements OnInit, AfterViewInit {
...
+          this.updateDateRange(moment(Number(queryParams.timestampStart)).toDate(), moment(Number(queryParams.timestampEnd)).toDate())
+          this.store.dispatch(new ExecutionEventsActions.Get(queryParams))
+          this.store.dispatch(new DashboardFormActions.InitializeForm({ minDate: this.timestampStart, maxDate: this.timestampEnd }))
+        } else {
+          this.store.dispatch(new ExecutionEventsActions.GetDefault({}))
+        }
       })
 
     this.store
       .select('dashboardFilters', 'dashboardFilters', 'value', 'dateRange')
```
[the remark this answers is not in the document]
Comment (leborgneromain) [reply client-ui/src/app/components/dashboard/dashboard.component.ts]: dashboardFilters is the name of the form as well, I will change it bc it's a bit weird indeed

File: client-ui/src/app/components/lineage-overview/lineage-overview.component.html [hunk 3]
```diff
@@ -20,8 +20,15 @@
...
+          width="40px" height="40px" style="margin-left:20px" />
```
[the remark this answers is not in the document]
Comment (leborgneromain) [reply client-ui/src/app/components/lineage-overview/lineage-overview.component.html]: No... you right, inlined styles should not be here. We should use classes instead

File: client-ui/src/app/components/lineage-overview/lineage-overview.component.html [hunk 4]
```diff
@@ -20,8 +20,15 @@
...
...Q0DayDBKlUnmIICPZQw2zFEXcssRpDqDgt2u0oZd5K4cP2uf0A11A2Sj1LDoml4Q3Q86tcUBgz1gZw3HipX/MIUT029HL822bJZtVC60rOHpkOQc8dD5fKnJBbx2vRXkP1itX4HMoi8OVC10l6cfRL5lJSlWJm+SDi6FtE7SuusMVC5yjqtbAmACOYnjqzC0KcfgFHP26FCHdF2g/Uj20gkQl+777guADEAqbI1S3YUzC0cqEYaUVwrRm2XeJkwPmkCwC6SW0ibMFfpIxY4SkWp9YwZmcCIVOGQVAFgQsBUykLTaVCsOWWoF9RwyFZyqZuEpjYupQsfTjpN7eCUEmCZG9TrE9YQBiMYuyiSedM3oQbKBLpj2iEQbyw1cYx0tjBcKLePwRfqBV/xGcPCzZ0SASCiNj66BJWOaBEEwXtBvCCAqgCG+wEVBbJKp0Lpor1Jwafp0DIMfXac5fNJXj+w7l2DcBgQgklToPTmXUmBiSeoer+nHLha8jdQIkocEc0gxAoikDAoIRb4QAmRWZO/DKWLnKlT1PjPy1BprWXFMchCx5Jm6a+B4Gp2eGra5G6xSDN0SvRuygWAgKrfewOqve3ghEeMCQM9JBCUGk1/ma1CAjm7QNuqyYiFGP2rGVLod9pNyDSNVIiJlT7yapU66G+0VIuoFsNAEKjWbZeBdsenhQAMqIpbZqCUpW2rZggtQgRNMFArYhUTksEDGSWI6/QPyZH1RwVAhDMJ3qFgSZns4EnMULIIquEwhY6ADcYlWtxY18ycabUDDAacVKpgw1sg5xRq27aFi6IZQjj9LvXu9VClzmU/TVsaR/c8p1F3fr7A1pkoeTysNqFonTznoeDPyI4ZcmqRRy53PFQsP5rWzB82AhAjfd2uq5F0bFakMo12BWatJA0kHASydJr1yPFFy7030l5C9VAPbJmBTC3z+aNVND+vi7tHMq3uYawaKmkOkjcbqu56YVjSclgCHc/chv03Y+R0i2ZLILWNfOeDKmxavT8MqJSkPbDgSlItQwCSB6iuT/my4R/KlhECEC8QjY/eikpHQ/R/00ah1TNm/1GQCvOlx6OO2pfNIpZVIN8DUxejl0/fxC9B2eLMC2DNSAEQ2f6bf7YKYzsDWQhKb0pvk60ITlbeGbECYIVYmThmVAAykasmcPo/JomOFwQTjDMAAAAASUVORK5CYII="
```
[the remark this answers is not in the document]
Comment (leborgneromain) [reply client-ui/src/app/components/lineage-overview/lineage-overview.component.html]: true

File: client-ui/src/app/components/lineage/lineage-graph/lineage-graph.component.ts [hunk 5]
```diff
@@ -71,23 +73,32 @@ export class LineageGraphComponent implements OnInit, AfterViewInit {
           this.cytograph.cy.layout(state.layout).run()
         }
       })
+    this.store.select('lineageOverview').subscribe(state => {
+      this.currentLineageOverview = state
```
[the remark this answers is not in the document]
Comment (leborgneromain) [reply client-ui/src/app/components/lineage/lineage-graph/lineage-graph.component.ts]: yes we can do it that way as well

**Pull request merged**